arch/riscv/core/swi_tables.ld

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2025 Google
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#if LINKER_ZEPHYR_FINAL && defined(CONFIG_ISR_TABLES_LOCAL_DECLARATION)
+INCLUDE isr_tables_swi.ld
```

Comment:
Style: seems weird to use a C preprocessor construct to gate the use of a GNU ld "INCLUDE" line vs. just using the natural #include which we do everywhere else.   Seems like a recipe for confusion, as we'd need to be sure the linker itself is run with the same set of include paths as its generator.

I guess this crept in in the original ISR_TABLES_LOCAL_DECLARATION feature, so not a -1 if there's desire to keep the implementations consistent.  But IMHO both should be #include.

Reply:
Good catch. How about I create a followup PR that changes both implementations to use `#include`?

Reply:
@andyross you good with this proposal or the PR as-is?

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/91385